CopyQ.py

```diff
@@ -69,7 +69,8 @@ def handleQuery(query):
                 )
             ]
 
-        script = copyq_script_getMatches % query.string if query.string else copyq_script_getAll
+        escaped_query = query.string.replace('\\', '\\\\').replace('"', '\\"')
```

Comment:
why is this necessary?

Reply:
Otherwise the `script` will not have valid js syntax if the query includes `"` or `\\`.